gslib/commands/requesterpays.py

```diff
@@ -71,7 +71,7 @@
 
 _GCLOUD_FORMAT_STRING = ('--format=value[separator=": "](' + 'name.sub("' +
                          shim_util.get_format_flag_caret() + '", "gs://"),' +
-                         'billing.requesterPays.yesno("Enabled", "Disabled"))')
```

Comment:
Nope, Both are different PRs with different fixes.

Reply:
I think you misunderstood. I understand this PR is for a different purpose, but I can see that this exact change is present in the other PR as well. Please check for the same file in the "Files Changed" tab.

Reply:
You can remove this now as well ? Why wait ?

Reply:
I want to keep change in this PR because the other one contains all the shim commands which requires --raw flag removal. I will remove it from other one.